wot/integration/src/main/java/org/eclipse/ditto/wot/integration/DittoWotIntegration.java

```diff
@@ -78,6 +81,13 @@ public interface DittoWotIntegration extends Extension {
      */
     WotThingModelValidator getWotThingModelValidator();
 
+    /**
+     * Updates the dynamic validation configs and recreates the validator with the new configs.
+     *
+     * @param configs the new dynamic validation configs
+     */
+    void updateDynamicConfigs(Set<TmValidationConfig> configs);
```

Comment:
You should accept a `Collection` instead of a `Set` - this makes the signature more flexible - and you can also paste in a `List`.

In the implementation, you can (and should) still wrap it in a new HashSet.

Reply:
`@since` is missing